feature-libs/asm/occ/adapters/occ-asm.adapter.ts

```diff
@@ -40,9 +69,16 @@ export class OccAsmAdapter implements AsmAdapter {
       true,
       new HttpHeaders()
     );
-    let params: HttpParams = new HttpParams()
-      .set('baseSite', this.activeBaseSite)
-      .set('sort', 'byNameAsc');
+    let params: HttpParams = new HttpParams().set(
+      'baseSite',
+      this.activeBaseSite
+    );
+
+    if (typeof options['sort'] !== 'undefined') {
+      params = params.set('sort', '' + options.sort);
+    } else {
+      params = params.set('sort', 'byNameAsc');
+    }
```

Comment:
can be this `params = params.set('sort', options.sort ?? 'byNameAsc');`.

Reply:
This suggestion is not adequate.  There are 3 cases here:
1. Sort is provided
   - use provided value 
1. Sort not provided, normal customer search
   - use default value
1. Sort not provided, customer list search
   - do not send default, let api select default value
  
The valid sort values for a customer list search are defined per the search strategy used for the list.  The sort value for a normal customer search is highly likely to not be valid.  Unless it is explicitly specified, the sort value is not sent so that the API can respond with its default sort method.  The chosen sort value is readable in the response should you need to make UI adjustments.

As a side note, I don't believe we should be providing a hard-coded default for sort for any case.